MapboxNavigation/RouteVoiceController.swift

```diff
@@ -61,6 +54,24 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     
     
     /**
+     If true, a noise indicating the user is going to be rerouted will play prior to rerouting.
+     */
+    public var playRerouteSound = false
+
+    
+    /**
+     Sound to play prior to reroute. Inherits volume level from `volume`.
+     */
+    public var rerouteSound = NSDataAsset(name: "reroute-sound", bundle: Bundle.navigationUI)
```

Comment:
A developer who wants to customize this sound would add the sound to their asset catalog and refer to it by `NSDataAsset(name: "oh", bundle: .main)`. That seems pretty reasonable, but it does require using an asset catalog. In other words, it precludes the traditional workflow of dragging an audio file into the project (thereby adding it to the resource bundle directly) and creating some other type from a path. NSDataAsset seems more robust, in any case.